lib/functions/rootfs/distro-agnostic.sh

```diff
@@ -400,11 +400,11 @@ function install_distribution_agnostic() {
 	if [[ "${BSPFREEZE:-"no"}" == yes ]]; then
 		display_alert "Freezing Armbian packages" "$BOARD" "info"
 		chroot_sdcard apt-mark hold "${image_artifacts_packages["armbian-plymouth-theme"]}" "${image_artifacts_packages["armbian-zsh"]}" \
-		"${image_artifacts_packages["armbian-config"]}" "${image_artifacts_packages["armbian-bsp-desktop"]}" \
-		"${image_artifacts_packages["armbian-desktop"]}" "${image_artifacts_packages["armbian-bsp-cli"]}" \
-		"${image_artifacts_packages["armbian-firmware"]}" "${image_artifacts_packages["armbian-firmware-full"]}" \
-		"${image_artifacts_packages["linux-headers"]}" "${image_artifacts_packages["linux-dtb"]}" \
-		"${image_artifacts_packages["linux-image"]}" "${image_artifacts_packages["uboot"]}" || true
+			"${image_artifacts_packages["armbian-config"]}" "${image_artifacts_packages["armbian-bsp-desktop"]}" \
```

Comment:
Curious on why your editor doesn't pick up the .editorconfig and ends up changing this....

Reply:
This is output of shellfmt. Tabbing seems right to me since it's just a part of chroot

Reply:
You're absolutely right, I'm delirious / my fork had this already.